src/core/text/Text.js

```diff
@@ -246,11 +183,14 @@ Object.defineProperties(Text.prototype, {
 
 /**
  * Renders text and updates it when needed
- *
+ * @param respectDirty {boolean} Whether to abort updating the text if the Text isn't dirty and the function is called.
  * @private
  */
-Text.prototype.updateText = function ()
+Text.prototype.updateText = function (respectDirty)
```

Comment:
I believe the expectation for developers with `updateText()` is that it forces a redraw regardless of `dirty`. I think this is a much more typical use-case. Without changing the call signature of `updateText`, another simpler approach is to make an internal update method which is used instead of the `updateText(true)`:

```js
Text.prototype.safeUpdateText = function() {
    if (this.dirty) {
        this.updateText();
    }
};
```